_layouts/downloadpage.html

```diff
@@ -42,34 +43,36 @@ <h3>Then, install Scala:</h3>
+                <span>Download IntelliJ</span>
               </a>
               <ul>
-                {% include tutorial-list.html column=1 %}
+                <li><a href="/documentation/getting-started-intellij-track/getting-started-with-scala-in-intellij.html"><i class="fa fa-file-text-o"></i>Getting Started with Scala in IntelliJ</a></li>
+                <li><a href="/documentation/getting-started-intellij-track/building-a-scala-project-with-intellij-and-sbt.html"><i class="fa fa-file-text-o"></i>Building a Scala Project with IntelliJ and sbt</a></li>
+                <li><a href="/documentation/getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.html"><i class="fa fa-file-text-o"></i>Testing Scala in IntelliJ with ScalaTest</a></li>
               </ul>
             </div>
             <div class="download-sbt">
               <div class="description">
                 <img src="/resources/img/download/arrow-right.png" alt="">
                 <p>Best if you are familiar with the command line</p>
               </div>
-              <a href="" class="btn-download" id="download-sbt-link">
+              <a href="/documentation/getting-started-sbt-track/getting-started-with-scala-and-sbt-on-the-command-line.html" class="btn-download" id="download-sbt-link">
                 <i class="fa fa-download"></i>
-                <span>Download Sbt</span>
+                <span>Get started with sbt</span>
               </a>
               <ul>
-                {% include tutorial-list.html column=0 %}
+                <li><a href="/documentation/getting-started-sbt-track/getting-started-with-scala-and-sbt-on-the-command-line.html"><i class="fa fa-file-text-o"></i>Getting Started with Scala and sbt on the Command Line</a></li>
+                <li><a href="/documentation/getting-started-sbt-track/testing-scala-with-sbt-on-the-command-line.html"><i class="fa fa-file-text-o"></i>Testing Scala with sbt and ScalaTest on the Command Line</a></li>
               </ul>
             </div>
           </div>
 
           <p class="bottom-lead">
-            Compared to other programming languages, installing Scala is a bit unusual. Scala is unusual because it is usually installed for each of your Scala projects rather than being installed system-wide. Both of the above options manage (via sbt) a specific Scala version per Scala project you create.
-            <br><br>
-            But it's also possible to "install" Scala in numerous other ways; e.g., grab Scala binaries and use Scala from the command line or use Scala in your browser!
-            <br><br>
+            Compared to other programming languages, installing Scala is a bit unusual.
+            In addition to the system-wide installation mentioned above, build tools such as sbt will automatically manage a specific Scala version per Scala project you create.
+            This makes sure that there are no version conflicts between different Scala projects on your computer.
```

Comment:
> This makes sure that there are no version conflicts between different Scala projects on your computer

Perhaps: "This allows each Scala project on your computer to use any version of Scala it wants, without interfering with each other."